src/main/java/org/jenkinsci/plugins/workinghours/model/ExcludedDate.java

```diff
@@ -121,99 +87,119 @@ public Boolean shouldExclude(Calendar date) {
         return false;
     }
 
-    public static class DataContainer {
+    /**
+     * Minutes offset to the UTC time, indicates the base timezone of the excluded date.
+     * Default to UTC(UTC+0).
+     */
+    private int utcOffset = 0;
 
-        /**
-         * Minutes offset to the UTC time, indicates the base timezone of the excluded date.
-         * Default to UTC(UTC+0).
-         */
-        public int utcOffset = 0;
+    /**
+     * Name of the selected timezone.
+     */
+    private String timezone;
 
-        /**
-         * A type that indicates whether
-         */
-        public String type = DateType.TYPE_GREGORIAN;
+    /**
+     * A type that indicates whether
+     */
+    private String type = DateType.TYPE_GREGORIAN;
 
-        /**
-         * The display name of this rule.
-         */
-        public String name = "";
+    /**
+     * The display name of this rule.
+     */
+    private String name = "";
```

Comment:
I suspect this isn't necessary now that the class no longer extends `AbstractDescribableImpl`. Please remove

Reply:
Actually this field is in use, this is for each excluded date's name, not the old one in descriptor.